src/views/Proposals/Community.tsx

```diff
@@ -68,72 +57,61 @@ const Community = (props: any) => {
     }, []);
 
     const handleAddCommunity = async () => {
-        setIsLoading(true);
+        try {
+            setIsLoading(true);
 
-        const response = await addCommunity({
-            ...contract,
-            ambassador: ambassadorAddress,
-            decreaseStep: toToken(0.01),
-            managers: [requestByAddress],
-            maxTranche: toToken(5, { EXPONENTIAL_AT: 25 }),
-            minTranche: toToken(1),
-            proposalDescription: `## Description:\n${description}\n\nUBI Contract Parameters:\nClaim Amount: ${toNumber(
-                contract.claimAmount
-            )}\nMax Claim: ${toNumber(
-                contract.maxClaim
-            )}\nBase Interval: ${frequencyToText(
-                contract.baseInterval
-            )}\nIncrement Interval: ${
-                (contract.incrementInterval * 5) / 60
-            } minutes\n\n\nMore details: ${config.baseUrl}/communities/${id}`,
-            proposalTitle: `[New Community] ${name}`
-        });
+            const response = await addCommunity({
+                ...community.contract,
+                ambassador: community.ambassadorAddress,
+                decreaseStep: toToken(0.01),
+                managers: [community.requestByAddress],
+                maxTranche: toToken(5, { EXPONENTIAL_AT: 25 }),
+                minTranche: toToken(1),
+                proposalDescription: `## Description:\n${community.description}\n\nUBI Contract Parameters:\nClaim Amount: ${toNumber(
+                    community.contract.claimAmount
+                )}\nMax Claim: ${toNumber(
+                    community.contract.maxClaim
+                )}\nBase Interval: ${frequencyToText(
+                    community.contract.baseInterval
+                )}\nIncrement Interval: ${
+                    (community.contract.incrementInterval * 5) / 60
+                } minutes\n\n\nMore details: ${config.baseUrl}/communities/${community.id}`,
+                proposalTitle: `[New Community] ${community.name}`
+            });
```

Comment:
Are all those fields required in the forms?

Reply:
According to Bernardo yes.
This is the same as the example he gave me